InvenTree/InvenTree/settings.py

```diff
@@ -1000,6 +1000,11 @@
 
 SOCIALACCOUNT_STORE_TOKENS = True
 
+# Explicitly set empty URL prefix for OIDC
+# The SOCIALACCOUNT_OPENID_CONNECT_URL_PREFIX setting was introduced in v0.60.0
+# Ref: https://docs.allauth.org/en/latest/release-notes/recent.html#backwards-incompatible-changes
```

Comment:
```suggestion
# Ref: https://github.com/pennersr/django-allauth/blob/0.60.0/ChangeLog.rst#backwards-incompatible-changes
```

I would prefer to use a permanent reference, as the release notes on docs.allauth.org do not allow to point to the changes of a specific release